eng/pipelines/scripts/generate-project-list.ps1

```diff
@@ -25,6 +25,7 @@ if ($env:ADDITIONALMODULESJSON -and $env:ADDITIONALMODULESJSON -notlike '*Additi
 # which means this is running as part of the pullrequest pipeline and the project list needs to
 # be figured out from the packageInfo files.
 if ($projectList.Length -eq 0 -and $ENV:PACKAGEINFODIR) {
+  $packageInfoFiles = $()
```

Comment:
```suggestion
  $packageInfoFiles = @()
```

I assume you want this to be an array?

Reply:
Yes. I've pushed another commit with this update. It's odd that powershell didn't complain about that when running locally or in the pipeline.